src/OpenTracing/ISpan.cs

```diff
@@ -1,164 +1,135 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 
 namespace OpenTracing
 {
+    using System;
```

Comment:
should be outside of the namespace as well :)

Reply:
acck I had to reset my settings for work, this should be easier :-)